bin/gobuild.sh

```diff
@@ -53,7 +53,7 @@ fi
 # at the beginning of the build and used throughout
 if [[ -z ${BUILDINFO} ]];then
```

Comment:
please undo the mode change to 644.  Executables should be 755.

Cheers
-steve

Reply:
OOPS... I developed it on Windows platform and the file mode was changed automatically.